src/components/PageHeadingApiKeys.vue

```diff
@@ -2,7 +2,7 @@
   <page-heading class="page-heading-api-keys" :crumbs="crumbs">
     <template #after-crumbs>
       <!-- Note: This doesn't do anything yet -->
-      <p-button inset size="xs" icon="PlusIcon" />
+      <p-button small icon="PlusIcon" />
```

Comment:
I wonder if this component is even used judging by the comment for this button

Reply:
I'm sure it's not. As I review each component individually (once I get a build working), I might try and check the usage of a lot of these components and see what can be deleted.